tests/model_registry/model_catalog/utils.py

```
@@ -243,3 +243,33 @@ def validate_model_catalog_configmap_data(configmap: ConfigMap, num_catalogs: in
+
+    Returns:
+        Dictionary containing the API response
+    """
+    url = f"{model_catalog_rest_url[0]}models?pageSize={page_size}"
+
+    if source_label:
+        url += f"&sourceLabel={source_label}"
+
+    url += additional_params
+
+    return execute_get_command(url=url, headers=model_registry_rest_headers)
```

Flake8 (7.3.0)

[error] undefined name 'execute_get_command'

(F821)